.github/workflows/assign-urgency-to-issue.yml

```diff
@@ -72,9 +72,11 @@ jobs:
       - name: Dump issue JSON
         if: always()
         run: |
-          ISSUE_JSON='${{ toJSON(github.event.issue) }}'
-          echo 'Github Issue Context JSON:'
-          printf '%s\n' "$ISSUE_JSON"
+          set +e +u +o pipefail
```

Comment:
Why do we need either of these `set` options?
There's no variables later (`u`), no pipes (`+o pipefail`). `+e` may be useful if you want to ignore falures, but this part with multiline `cat` does not have a place to fail.

My recommendation would be to keep it lean (even `if:` is not needed here, same for masking exit code with `exit 0`):

```yaml
      - name: Dump issue JSON
        run: |
          cat << 'EOF'
          ${{ toJSON(github.event.issue) }}
          EOF
```